packages/curl/build

```diff
@@ -4,6 +4,9 @@ export CPPFLAGS=-I/opt/mesosphere/include
 export LDFLAGS="-L/opt/mesosphere/lib -Wl,-rpath=/opt/mesosphere/lib"
 export LD_LIBRARY_PATH=/opt/mesosphere/lib
 
+# just so that first test does not yield failed directory check.
+mkdir -p /var/lib/dcos/pki/CA/trusted-certs
```

Comment:
Will revisit naming once we converge towards a solution.

Agree with what you say, but here I think prior art / "standard" is one of the shittiest shit shows: https://www.happyassassin.net/2015/01/12/a-note-about-ssltls-trusted-certificate-stores-and-platforms/ (that one is really worth a read).

If we end up having a directory containing individual trusted root CA certificate files and where we expect people to add custom root CA certificates, then I tend to prefer to give this directory a name different from just "certs".